code/modules/research/xenobiology/crossbreeding/unstable.dm

```diff
@@ -0,0 +1,283 @@
+
+/obj/item/slimecross/unstable/red
+	colour = "red"
+	effect_desc = "When thrown causes extreme bleeding to the first person hit."
+
+/obj/item/slimecross/unstable/red/on_effect(atom/hit_atom)
+	if(!ishuman(hit_atom))
+		new /obj/effect/decal/cleanable/blood(get_turf(hit_atom))
+		return
+	var/mob/living/carbon/human/humie = hit_atom
+	humie.bleed_rate += 15
+
+/obj/item/slimecross/unstable/red/on_effect(atom/hit_atom)
+	if(!ishuman(hit_atom))
+		new /obj/effect/decal/cleanable/blood(get_turf(hit_atom))
+		return
+	var/mob/living/carbon/human/humie = hit_atom
+	humie.bleed_rate += 15
+
+/obj/item/slimecross/unstable/green
+	colour = "green"
+	effect_desc = "When thrown causes 25 brute damage, you heal for the amount of damage it deals after armor mitigation."
+
+/obj/item/slimecross/unstable/green/on_effect(atom/hit_atom)
+	if(!isliving(hit_atom))
+		return
+
+	var/mob/living/livie = hit_atom
+	var/health_amt_now = livie.getBruteLoss()
+	livie.apply_damage(25,BRUTE)
+	var/health_to_heal = health_amt_now - livie.getBruteLoss()
+	human_thrower.adjustBruteLoss(health_to_heal)
+
+/obj/item/slimecross/unstable/pink
+	colour = "pink"
+	effect_desc = "When thrown it hugs the first thing it hits."
+
+/obj/item/slimecross/unstable/pink/on_effect(atom/hit_atom)
+	if(!iscarbon(hit_atom))
+		return
+	var/mob/living/carbon/carbie = hit_atom
+	human_thrower.help_shake_act(carbie)
+
+/obj/item/slimecross/unstable/gold
+	colour = "gold"
+	effect_desc = "When thrown spawns goliath tentacles."
+
+/obj/item/slimecross/unstable/gold/on_effect(atom/hit_atom)
+	if(isliving(hit_atom))
+		var/mob/living/livies = hit_atom
+		livies.Stun(40)
+	new /obj/effect/temp_visual/goliath_tentacle/original(get_turf(hit_atom),human_thrower)
+
+/obj/item/slimecross/unstable/oil
+	colour = "oil"
+	effect_desc = "When thrown it explodes."
+
+/obj/item/slimecross/unstable/oil/on_effect(atom/hit_atom)
+	dyn_explosion(get_turf(hit_atom),0.5)
+
+/obj/item/slimecross/unstable/black
+	colour = "black"
+	effect_desc = "When thrown it stains all visor in a 3x3 radius of the impact, additionally it also produces a small smoke cloud, both effects dont interact with eachother."
+
+/obj/item/slimecross/unstable/black/on_effect(atom/hit_atom)
+	var/datum/effect_system/smoke_spread/smoke = new
+	smoke.set_up(2, get_turf(hit_atom))
+	smoke.start()
+	for(var/mob/living/carbon/human/humie in range(get_turf(hit_atom),2))
+		var/obj/item/clothing/glasses/big_g = humie.glasses
+		if(!big_g)
+			return
+		big_g.AddElement(/datum/element/dirty_glasses)
+
+/obj/item/slimecross/unstable/adamantine
+	colour = "adamantine"
+	effect_desc = "When thrown craetes deadly adamantine shards on the floor that rip apart shoes."
+
+/obj/item/slimecross/unstable/adamantine/on_effect(atom/hit_atom)
+	new /obj/item/slimecross/unstable/adamantine(get_turf(hit_atom))
+	for(var/i in 1 to 4)
+		new /obj/item/shard/adamantine(get_step(hit_atom,pick(GLOB.alldirs)))
```

Comment:
wouldn't it be better to use `range` and spawn them consistently? Or is the unpredictable stacking intentional/preferred?

Reply:
an element of randomness is more fun than predictable patterns.

Reply:
I suppose. I just thought it would have looked a bit weird, having all of them spawn in the top left tile, etc.